src/PK/CommandExtraBundle/DependencyInjection/Configuration.php

```diff
@@ -10,6 +10,9 @@
  */
 class Configuration implements ConfigurationInterface
 {
+    /**
+     * @inheritdoc
+     */
```

Comment:
Could you change the default pid_dir value here to the Symfony3 `var/run` directory (instead of the current `app/var/run`)?